src/Microsoft.Health.Dicom.Core/Configs/AuthenticationConfiguration.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.Health.Dicom.Core.Configs;
 
 public class AuthenticationConfiguration
 {
+    public string Name { get; set; }
```

Comment:
Could we just make use of the `authority` in this case instead of specifying a `name`?

Reply:
Make sense. Name is only to identify which scheme should be used, so authority works here too